### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Xi is principally tested.
 ### Theme
 
 A few theme files are bundled with the application. A theme can be selected
-from the Debug > Theme menu. There is not yet a mechanism for including
+from the `Debug` > `Theme menu`. There is not yet a mechanism for including
```
Comment (cmyr) [inline README.md]: The word 'menu' should be outside of the ticks.

File: README.md [hunk 2]
```diff
@@ -50,20 +50,20 @@ as a subdirectory.
...
+If you've installed with [rustup](https://rustup.rs), make sure Rust is up to date by running
 `rustup update stable`.
 
 
 ## Configuration
 
 User settings are currently stored in files; the general preferences are
 located at `~/Library/Application Support/XiEditor/preferences.xiconfig`.
-This file can be opened from File > Preferences (⌘ + ,).
+This file can be opened from `File` > `Preferences (⌘ + ,)`.
```
Comment (cmyr) [inline README.md]: Mentioned in your other PR; you can use the markup `<kbd>⌘</kbd>` to have this rendered as a keycap: <kbd>⌘</kbd>.

**Comment by cmyr** [comment]
@kant are you interested in fixing this up?

**Comment by raphlinus** [comment]
Closing due to low importance and lack of response.

**Pull request closed without merging**